packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-base64.yaml

```diff
@@ -0,0 +1,6 @@
+Resources:
+  Base64Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName:
+        !Base64 NonBase64BucketName
```

Comment:
Can you rename all these files to have the prefix `short-form` instead of just `short`?